src/osd/PrimaryLogPG.cc

```diff
@@ -14109,7 +14109,7 @@ uint64_t PrimaryLogPG::recover_backfill(
 	dout(20) << " BACKFILL keeping " << check
 		 << " with ver " << obj_v
 		 << " on peers " << keep_ver_targs << dendl;
-	//assert(!waiting_for_degraded_object.count(check));
+	//ceph_assert(!waiting_for_degraded_object.count(check));
```

Comment:
A commented out assert could probably just be removed.

Reply:
Please see followup comment below regarding cleanups.